Makefile

```diff
@@ -66,7 +59,8 @@ endif
 golib: golib-clean
 	@docker run --rm -v $(CURDIR):/sensorbucket --user `id -u` \
 		openapitools/openapi-generator-cli:v6.2.1 generate -i /sensorbucket/tools/openapi/api.yaml \
-		-g go -o /sensorbucket/pkg/api -t /sensorbucket/tools/openapi-templates/go \
+		-g go -o /sensorbucket/pkg/api \
```

Comment:
Because I have downgraded the OpenAPI Generator due to a regression issue. As a consequence the templates are not the same and I didn't want to spent time optimizing the templates for a temporary issue. Once the generator version can be set back to latest, templates can be optimized.